examples/scroll-sync.amp.html

```diff
@@ -22,7 +22,10 @@
   </div>
 
   <div>
-    <amp-fx-scroll-sync name='scale'>
+    <amp-fx-scroll-sync name='scale'
```

Comment:
I see. You are right. Not sure what I was thinking. `end-position` is certainly cool.

What I was thinking, however, is that predicting "start-position" might be hard. E.g. start position would be most likely the position of docking or such. With responsive layout, fonts and other things, it might be hard to predict when exactly that would happen. So:

- start position is somehow triggered
- end position is defined as start position + "scale over X px".